### DISCUSSION TITLE
Quotient familial : demie-part invalidité

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2639] [Idées]
Demandée par un utilisateur

Bofip : https://bofip.impots.gouv.fr/node/3451

Connexe sur le calcul du quotient familial #1247

**Comment by glopezrios** [comment]
Un utilisateur précise ce manque. Afin de calculer avec justesse le nombre de part fiscal du foyer, il manque dans le simulateur « auto-entrepreneur » deux questions sur :
-	le nombre d’enfants handicapés
-	le nombre d’adultes invalides à charge.

Ces deux questions permettraient d’intégrer [les majorations de part fiscal dans ces situations](https://www.monparcourshandicap.gouv.fr/actualite/declaration-des-revenus-et-handicap-que-faut-il-savoir). Ces questions, couplées à celles existantes sur le nombre d’enfants et la situation familiale, permettrait de mieux définir le nombre de part fiscal du foyer et donc le montant de l’impôt.

**Comment by JalilArfaoui** [comment]
À prolonger dans #2998
